javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,62 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === '' || studentName === undefined || studentName === null) {
```

Comment:
There is no need to check for undefined and null.

Reply:
okay,noted
Thank You.